### ISSUE TITLE
Following the accessibility audit create accessibility tickets

### ISSUE THREAD

**Issue by christinach** [pulibrary/static-tables#77]
Following the [accessibility audit](https://github.com/pulibrary/static-tables/issues/14) create accessibility tickets

Acceptance Criteria:
- [x] Use the [accessibility ticket template](https://github.com/pulibrary/static-tables/issues/new?assignees=&labels=accessibility-general%2Cinvestigate&projects=&template=accessibility_audit.md&title=Audit+%5BWhich+part+of+the+project%3F%5D+for+accessibility)
- [x] Tickets should have one of the accessibility labels

**Comment by sandbergja** [comment]
Thanks, @christinach !  Looks great 🐬
